modules/vba_documents/app/workers/vba_documents/upload_remover.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'vba_documents/object_store'
+
+module VBADocuments
+  class UploadRemover
+    include Sidekiq::Worker
+
+    def perform
+      return unless Settings.vba_documents.s3.enabled
+      VBADocuments::UploadSubmission.where("status = 'received' AND s3_deleted IS NOT True").find_each do |upload|
```

Comment:
I think this is likely to not cover all the cases:
- object transitions from `received` to `processing` in interval before this job runs
- object is uploaded, goes to `received`, we delete it, but s3 upload URL is still valid and somebody uploads another payload (which they should not do, but...)

I think I'd like to proceed more like the Pension/Burial worker that finds records with `created_at older than <X>` and `status != pending || uploaded` and cleans them up. Where <X> is like 3 days. That would give us some time to inspect/debug any problematic submissions as well. 

Should continue to use the s3_deleted column so we don't have to scan an ever-growing number of records.